judge/migrations/0088_compiler_message_ansi2html.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 2.1.12 on 2019-09-17 13:30
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import lxml.html as lh
+from lxml.html.clean import clean_html
+
+
+def strip_error_html(apps, schema_editor):
+    Submission = apps.get_model('judge', 'Submission')
+    for sub in Submission.objects.filter(error__isnull=False):
```

Comment:
You would use [`QuerySet.iterator`](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#iterator) with a large chunk size if you cared about this not being insanely slow and use up all the memory.